### PR TITLE
Add integration tests for dispatch and service after running server

### PR DESCRIPTION
**Pull request by aqt01**

### CODE REVIEW

File: cmd.sh [hunk 1]
```diff
@@ -24,28 +24,65 @@ set -o nounset
...
+		nohup pocket-core --dispatch \
+		  --datadirectory ${POCKET_PATH_DATADIR:-datadir} \
+		  --dbend ${POCKET_CORE_AWS_DYNAMODB_ENDPOINT:-dynamodb.us-east-1.amazonaws.com} \
+		  --dbtable ${POCKET_CORE_AWS_DYNAMODB_TABLE:-dispatch-peers-staging} \
+		  --dbregion ${POCKET_CORE_AWS_DYNAMODB_REGION:-us-east-1} \
+		  --disip ${POCKET_CORE_DISPATCH_IP:-127.0.0.1} \
+		  --disrport ${POCKET_CORE_DISPATCH_PORT:-8081} &
+
+		sleep 10 
+		exec go test ./tests/integration/... --dispatch --dispatchtesturl dispatch.staging.pokt.network --servicetesturl service.staging.pokt.network
```
Comment (luyzdeleon) [inline cmd.sh]: Can we pass the URL's as environment variables?

**Review by luyzdeleon — CHANGES_REQUESTED** [review]

File: cmd.sh [hunk 2]
```diff
@@ -55,7 +55,7 @@ if [ $POCKET_CORE_NODE_TYPE = "dispatch" ]; then
 		  --disrport ${POCKET_CORE_DISPATCH_PORT:-8081} &
 
 		sleep 10 
-		exec go test ./tests/integration/... --dispatch --dispatchtesturl dispatch.staging.pokt.network --servicetesturl service.staging.pokt.network
+		exec go test ./tests/integration/... --dispatch --dispatchtesturl ${POCKET_CORE_DISPATCH_IP:-127.0.0.1} --servicetesturl ${POCKET_CORE_DISPATCH_IP:-127.0.0.1}
```
Comment (luyzdeleon) [inline cmd.sh]: Shouldn't this be it's own env variable? and you used the same in both flags.

File: cmd.sh [hunk 3]
```diff
@@ -81,7 +81,7 @@ elif [ $POCKET_CORE_NODE_TYPE = "service" ]; then
 			--port ${POCKET_CORE_SERVICE_PORT:-8081} &
 		
 		sleep 10 
-		exec go test ./tests/integration/... --dispatchtesturl dispatch.staging.pokt.network --servicetesturl service.staging.pokt.network
+		exec go test ./tests/integration/... --dispatchtesturl ${POCKET_CORE_DISPATCH_IP:-127.0.0.1} --servicetesturl ${POCKET_CORE_DISPATCH_IP:-127.0.0.1}
```
Comment (luyzdeleon) [inline cmd.sh]: see above

**Review by luyzdeleon — CHANGES_REQUESTED** [review]

File: cmd.sh (hunk 1 above)
Comment (aqt01) [reply cmd.sh]: Oh, sorry. It's Done already

File: cmd.sh (hunk 2 above)
Comment (aqt01) [reply cmd.sh]: I think we have to tests the same variables that the server is using to connect

**Review by luyzdeleon — APPROVED** [review]

**Pull request merged**